ci-operator/config/openshift/cluster-api-provider-ibmcloud/openshift-cluster-api-provider-ibmcloud-main.yaml

```diff
@@ -0,0 +1,93 @@
+base_images:
+  base:
+    name: "4.9"
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
```

Comment:
just curious, do we need the 4.6 builders?

Reply:
idk - I was mimicking the content of the other provider docs.
Shall I remove lines 6-9?

Reply:
i don't think we do, since there will be no ibm cloud in 4.6 afaik. cc @JoelSpeed